emission/analysis/modelling/tour_model/similarity.py

```diff
@@ -77,8 +77,7 @@ def bin_data(self):
                 self.bins.append([a])
         self.bins.sort(key=lambda bin: len(bin), reverse=True)
 
-    #delete lower portion of bins
-    def delete_bins(self):
+    def calc_cutoff_bins(self):
         if len(self.bins) <= 1:
             self.newdata = self.data
             return
```

Comment:
What will return here?

Reply:
an empty return returns None
Note that I did not change that line